RZViewActions/UIView+RZViewActions.m

```diff
@@ -90,9 +97,16 @@ + (instancetype)action:(RZViewActionBlock)action withDuration:(NSTimeInterval)du
 
 + (instancetype)action:(RZViewActionBlock)action withOptions:(UIViewAnimationOptions)options duration:(NSTimeInterval)duration
 {
-    NSParameterAssert(action);
+    return [[RZViewAction alloc] initWithBlock:action options:options duration:duration];
```

Comment:
Wouldn't it just be `[self alloc]`? That's what was here before, and I changed it for a reason I can't remember.